albogdan/skule-vote
# Created react app called UI

## Overview

- Resolves #2 
- Also installed prettier, sass, and Material UI
- Added a "Test" section to the README and included commands to test FE. 


## Unit Tests Created

- N/A


## Steps to QA

- go to folder `skule-vote/skule_vote/frontend/ui` and run `yarn in && yarn start`. the React server will start on `localhost:3000` and you will see a grey page with "Skule Vote" in the middle
